docker/worker/Dockerfile

```diff
@@ -1,21 +1,18 @@
-FROM opensuse/leap:15.2
+#!BuildTag: openqa_worker
+FROM opensuse/leap:15.1
```

Comment:
should be fixed meanwhile by @perlpunk and me ensuring that devel:openQA:ci/base is triggered and built correctly, right?

Reply:
i think for testing that the meta file https://build.opensuse.org/package/meta/home:ilausuch:branches:devel:openQA/openQA_docker_images still needs to be changed to:
```
    <enable arch="x86_64" repository="containers15.2"/>

```

Reply:
Solved, see this in action https://build.opensuse.org/package/show/home:ilausuch:branches:devel:openQA/openQA_container_image_worker_x86